src/Module.ts

```diff
@@ -13,8 +13,7 @@ import extractNames from './ast/utils/extractNames';
 import enhance from './ast/enhance';
 import clone from './ast/clone';
 import ModuleScope from './ast/scopes/ModuleScope';
-import { encode } from 'sourcemap-codec';
-import { RawSourceMap, SourceMapConsumer } from 'source-map';
+import { RawSourceMap } from 'source-map';
```

Comment:
If we inline the `RawSourceMap` type we could remove `source-map` as a dependency entirely too.

Reply:
It's still used in tests, and it might be better to keep it there because it's a "reference" implementation, which is good for verifying source maps.